regression-test/suites/insert_p0/test_group_commit_partial_update.groovy

```diff
@@ -0,0 +1,128 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_group_commit_partial_update") {
+    def tableName = "test_group_commit_partial_update"
+
+    def getRowCount = { expectedRowCount ->
+        def retry = 0
+        while (retry < 30) {
+            sleep(2000)
```

Comment:
you can use  Awaitility.await()
Please refer to regression-test/suites//compaction/test_base_compaction.groovy

Reply:
<img width="1321" alt="image" src="https://github.com/user-attachments/assets/06218b52-de08-42d2-88bc-354e2442ccfd">